admin_guide/cluster_management.rst

```diff
@@ -42,6 +42,31 @@ The Citus master only stores metadata about the table shards and does not store
 
 However, in some write heavy use cases where the master becomes a performance bottleneck, users can add another master. As the metadata tables are small (typically a few MBs in size), it is possible to copy over the metadata onto another node and sync it regularly. Once this is done, users can send their queries to any master and scale out performance. If your setup requires you to use multiple masters, please `contact us <https://www.citusdata.com/about/contact_us>`_.
 
+Multi-tenancy
```

Comment:
The title here doesn't really give much information. I'd rather we have something more explicit, like "Isolating tenants in a multi-tenant table" (Or something like that).